### PR TITLE
Terminal resizing in the UI works for remote clusters

### LINKED ISSUE #778: Terminal resize calls are failing for remote cluster
## Description

When accessing remote cluster via UI, resize calls do not work.

Steps to reproduce:

* Create cluster A and B
* Access cluster B via proxy on cluster A
* Resize window.
* Resize call fails

The reason for this is that resize call goes to local proxy session, instead of using remote client session.

### PR DESCRIPTION
**Pull request by kontsevoy**
Fixes #778 

The PR may appear big because of all the docker stuff, but in reality I just added a proper processing of "site" URL argument and the back-end uses a different `auth.ClientI` pointer to talk to a proper auth API.

### CODE REVIEW

File: lib/web/sessions.go [hunk 1]
```diff
@@ -65,18 +65,19 @@ func (c *SessionContext) getTerminal(sessionID session.ID) (*terminalHandler, er
...
 
-	err := c.clt.UpdateSession(session.UpdateRequest{
+	err := siteAPI.UpdateSession(session.UpdateRequest{
 		ID:             sessionID,
 		TerminalParams: &params,
 		Namespace:      namespace,
 	})
 	if err != nil {
-		return trace.Wrap(err)
+		log.Error(err)
```
Comment (klizhentas) [inline lib/web/sessions.go]: I think I understand why fallthrough is necessary, but can you add comment for future readers?

Comment (kontsevoy) [inline lib/web/sessions.go]: well... it's not a critical error. no need to abort the function, quite common. frankly, I would even stick it into `_` and not even bother... I'll add a comment.

Comment (kontsevoy) [inline lib/web/sessions.go]: done

**Comment by klizhentas** [comment]
retest this please

**Pull request merged**